### ISSUE TITLE
Ensure spreadsheet methods work with different options

### ISSUE THREAD

**Issue by cliffckerr** [sciris/sciris#289]
E.g., `readcells()` should work if opened with pandas

**Comment by RomeshA** [comment]
Pandas now uses openpyxl as the backend by default, I'd be in favour of having `sc.Spreadsheet.wb` always be an `openpyxl.Workbook` and then to have

```
def pandas():
    return pd.ExcelFile(self.wb, engine='openpyxl')
```

**Comment by cliffckerr** [comment]
Closed by #323
